worker/issue-redirect/README.md

```
@@ -8,7 +8,10 @@ Target repo: `Embers-of-the-Fire/eve-fit-assistant`
 
 ### `POST /bug-report`
 
-Create a bug report issue.
+Create a bug report issue. The issue title is derived server-side as `[Bug]: <title>`, and
+the body mirrors the repository's GitHub bug-report issue form: `### `-level section
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)